build-scripts/bootstrap.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -e
+
+echo "Installing dependencies from brew.."
+# Install tools
+brew reinstall gradle flow watchman xctool
```

Comment:
I don't think `watchman` is required for production builds. Just monitoring JS files fir changes and re-bundling when in development.